pilot/proxy/envoy/policy.go

```diff
@@ -23,6 +23,16 @@ import (
 	"istio.io/istio/pilot/proxy"
 )
 
+func isDestinationBlacklistedForAuth(serviceName string, auth_blacklisted_services []string) bool {
```

Comment:
How about we use same language as the config? Like isDestinationExcludedForAuth

Reply:
Done